### PR TITLE
DOCS-220 add versions for IP blocking for Docker

### PR DESCRIPTION
**Pull request by AnastasiaTWW**

### CODE REVIEW

File: docs/admin-en/installation-docker-en.md [hunk 1]
```diff
@@ -175,4 +175,14 @@ docker exec -it wallarm-node /usr/lib/nagios-plugins/check_wallarm_export_delay
...
 
---8<-- "../include/installation-extra-steps.md"
\ No newline at end of file
+--8<-- "../include/installation-extra-steps.md"
+
+### Blocking Requests by IP Address
+
+The IP blocking functionality provides the following additional features:
+
+* If the WAF detects at least three different attack vectors from an IP address, the address is automatically added to the blacklist and blocked for 1 hour. If a similar behavior from the same IP address is detected again the IP is blocked for 2 hours, etc.
```
Comment (vgartvich-wallarm) [inline docs/admin-en/installation-docker-en.md:184]: Do we have this part describes anywhere else? The paragraph is relevant for all our node form-factors... Would you like to move it to a separate article?

File: docs/admin-en/installation-docker-en.md [hunk 2]
```diff
@@ -175,4 +175,14 @@ docker exec -it wallarm-node /usr/lib/nagios-plugins/check_wallarm_export_delay
...
+
+* Ability to use Wallarm to protect against behavior-based attacks such as [brute-force](../attacks-vulns-list.md#brute-force-attack), [path traversal attacks](../attacks-vulns-list.md#path-traversal) or [forced browsing](../attacks-vulns-list.md#forced-browsing).
+
+To enable IP blocking functionality, please select the configuration method at the [Methods of Blocking by IP Address](configure-ip-blocking-en.md) page and follow the appropriate instructions.
```
Comment (vgartvich-wallarm) [inline docs/admin-en/installation-docker-en.md:188]: We already have a good example formulated for the sidecar option...

File: docs/admin-en/installation-docker-en.md (hunk 1 above)
Comment (AnastasiaTWW) [reply docs/admin-en/installation-docker-en.md:184]: Yes, I think we need to extend this document https://docs.wallarm.com/admin-en/configure-ip-blocking-en/. Just a little later

File: docs/admin-en/installation-docker-en.md (hunk 2 above)
Comment (AnastasiaTWW) [reply docs/admin-en/installation-docker-en.md:188]: Yes, I copied the biggest part from https://docs.wallarm.com/admin-en/configuration-guides/wallarm-ingress-controller/best-practices/block-ip-addresses/. But the method of configuration is different and it's already described

**Pull request merged**